discovery/azure/azure.go

```diff
@@ -80,6 +80,13 @@ type SDConfig struct {
 	RefreshInterval model.Duration     `yaml:"refresh_interval,omitempty"`
 }
 
+func (c *SDConfig) validateAuthParam(param, name string) error {
```

Comment:
It doesn't need to be a receiver method so you can remove `(c *SDConfig)`.

Reply:
ok